Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/readme.md

```diff
@@ -2,65 +2,63 @@
 
```

Comment:
Why are you changing this readme.md file?

Reply:
android studio reformatted it for some reason